testdata/validate-data/data/value_type_violation.xml

```diff
@@ -74,7 +72,7 @@
 
     <resource label="Richtext" restype=":ClassWithEverything" id="id_richtext">
         <text-prop name=":testRichtext">
-            <text encoding="utf8">Text</text>
+            <text encoding="utf8"><p>Text</p></text>
```

Comment:
This should be richtext, this should not raise an error that the content is wrong.